### PR TITLE
New UT

### PR DESCRIPTION
**Pull request by oualib**
- vModels2
 - ML plots
 - ML repr
 - Some bugs corrections

### CODE REVIEW

File: verticapy/tests/utilities/test_utilities.py [hunk 1]
```diff
@@ -56,6 +56,11 @@ def world_vd(base):
 
 
 class TestUtilities:
+    def test_print_display(self, titanic_vd):
+        print(titanic_vd)
+        display(titanic_vd)
+        assert titanic_vd.shape() == (1234, 14)
```
Comment (afard) [inline verticapy/tests/utilities/test_utilities.py:62]: It doesn't seem that this function tests `print` or `display`. The `assert` line is irrelevant to the two lines above it.

File: verticapy/tests/vModel/test_knn_classifier.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+    def test_set_params(self, model):
+        model.set_params({"p": 1})
+
+        assert model.get_params()["p"] == 1
+
+    def test_model_from_vDF(self, base, titanic_vd):
+        model_test = KNeighborsClassifier("knn_from_vDF", cursor=base.cursor)
+        model_test.fit(titanic_vd, ["age"], "survived")
+        model_test.score(cutoff=0.9, method="accuracy") == pytest.approx(0.5691554467564259)
```
Comment (afard) [inline verticapy/tests/vModel/test_knn_classifier.py:172]: assert?

File: verticapy/tests/vModel/test_knn_regressor.py [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+    def test_set_params(self, model):
+        model.set_params({"p": 1})
+
+        assert model.get_params()["p"] == 1
+
+    def test_model_from_vDF(self, base, titanic_vd):
+        model_test = KNeighborsRegressor("knn_from_vDF", cursor=base.cursor)
+        model_test.drop()
+        model_test.fit(titanic_vd, ["age"], "survived")
+        model_test.score() == pytest.approx(0.5691554467564259)
```
Comment (afard) [inline verticapy/tests/vModel/test_knn_regressor.py:183]: assert?

File: verticapy/tests/vModel/test_nearestcentroid.py [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+    def test_set_params(self, model):
+        model.set_params({"p": 1})
+
+        assert model.get_params()["p"] == 1
+
+    def test_model_from_vDF(self, base, titanic_vd):
+        model_test = NearestCentroid("nc_from_vDF", cursor=base.cursor)
+        model_test.fit(titanic_vd, ["age"], "survived")
+        model_test.score(cutoff=0.9, method="accuracy") == pytest.approx(0.4312938816449348)
```
Comment (afard) [inline verticapy/tests/vModel/test_nearestcentroid.py:172]: assert

File: verticapy/tests/utilities/test_utilities.py (hunk 1 above)
Comment (oualib) [reply verticapy/tests/utilities/test_utilities.py]: Corrected - test deleted and one for the vdf repr was created

File: verticapy/tests/vModel/test_knn_classifier.py (hunk 2 above)
Comment (oualib) [reply verticapy/tests/vModel/test_knn_classifier.py]: Done;

File: verticapy/tests/vModel/test_knn_regressor.py (hunk 3 above)
Comment (oualib) [reply verticapy/tests/vModel/test_knn_regressor.py]: Done;

File: verticapy/tests/vModel/test_nearestcentroid.py (hunk 4 above)
Comment (oualib) [reply verticapy/tests/vModel/test_nearestcentroid.py]: Done;

**Review by afard — APPROVED** [review]

**Pull request merged**